product/views/ContainerGroup.yaml

```diff
@@ -41,6 +42,7 @@ col_order:
 - restart_policy
 - dns_policy
 - ext_management_system
+- ready_condition_status
```

Comment:
I think that for consistency (with nodes) it should go right after `container_project.name`